internal/command/logs/shipper_setup.go

```diff
@@ -147,40 +123,55 @@ func runSetup(ctx context.Context) (err error) {
 			CPUs:     1,
 			MemoryMB: 256,
 		},
-		Image: "flyio/log-shipper",
+		Image: "flyio/log-shipper:auto-d5a96e6",
 	}
 
 	machines, err := flapsClient.List(ctx, "")
 
+	if err != nil {
+		return err
+	}
+
 	launchInput := api.LaunchMachineInput{
```

Comment:
Should this and `machineConf ` be defined in the `else` block starting on L145?

Reply:
Done!